client/src/controllers/skip-link-controller.ts

```diff
@@ -0,0 +1,38 @@
+import AbstractController from './abstract-controller';
+
+/**
+ * Used to provide an accessible skip link for keyboard control so that
+ * users can easily navigate beyond the repeated content to the main content.
+ *
+ * Inspired by https://github.com/selfthinker/dokuwiki_template_writr/blob/master/js/skip-link-focus-fix.js
+ *
+ * @example
+ * <a class="button" href="#main" data-controller="w-skip-link">Skip to content</a>
```

Comment:
Just looking at this file, I was confused about how `skipToContent` was invoked, but then I saw the markup change down in `skeleton.html`. I think this example should be updated to match? Or keep it simplified, but at least add the `data-action` attribute?

```suggestion
 * <a class="button" href="#main" data-controller="w-skip-link" data-action="w-skip-link#skipToContent">Skip to main content</a>
```